lab1.ipynb

```diff
@@ -28,7 +28,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 35,
```

Comment:
Hm could you look into into if there are any best pratices for ipython notebooks in git repos? In the current form this feels closer to having an object file stored in the repo... (not a problem with this commit though, a more general question)

Reply:
Yes, I have put that onto my TODO list.